zipkin-collector-service/config/collector-cassandra.scala

```diff
@@ -40,6 +40,8 @@ if (username.isDefined && password.isDefined) {
   Factory.cassandraPassword.parse(password.get)
 }
 
+sys.env.get("CASSANDRA_MAX_CONNECTIONS").foreach(Factory.cassandraMaxConnections.parse(_))
```

Comment:
we don't have a great way of testing env variables at the moment, so I tested this via a debugger